api/admin/controller.py

```diff
@@ -168,6 +170,24 @@ def details(self, data_source, identifier):
         return entry_response(
             AcquisitionFeed.single_entry(self._db, work, annotator)
         )
+        
+    def complaints(self, data_source, identifier):
+        """Return detailed complaint information for admins."""
+        
+        pool = self.load_licensepool(data_source, identifier)
```

Comment:
I think that's better, but it really depends on how you handle a future case where there are 3 different editions of Frankenstein with different overlapping sets of complaints. But since it's the future you can mostly worry about it later.